bundle.Dockerfile

```diff
@@ -1,13 +1,17 @@
 FROM scratch
 
+LABEL com.redhat.openshift.versions="v4.8"
+LABEL com.redhat.delivery.backport=true
+LABEL com.redhat.delivery.operator.bundle=true
```

Comment:
why this now, at release time?

Reply:
This is most likely a result of bumping up controller-gen. Anyway, the bundle is built by RedHat. Any labels here will be replaced by those from `annotations.yaml`